src/QGCMapPalette.h

```diff
@@ -64,7 +64,7 @@ class QGCMapPalette : public QObject
     void lightColorsChanged(bool lightColors);
     
 private:
-    bool _lightColors;
+    bool _lightColors = false;
```

Comment:
Current QGC codebase doesn't use this style of member settings anywhere currently. It all uses constructor code style of setting these up. I'd rather not mix and match that usage pattern since it means you may have to look in two places to find what things are set to.

Reply:
Looks like there are a number of cases of this style already in the codebase. So fine for now.

Reply:
It seems more "right" to define the default right next to the definition instead of the constructor. It's easier to _not_ forget to initialize it and you don't have to pay attention to the initialization order, which the compiler complains if you do it in the wrong order within the constructor.

Reply:
I don't disagree with that. But the problem is that you can't do everything there. Hence you are left with some things in the constructor and some things in the header. Leaving you with two places to look for things.